modules/configuring-a-proxy-after-installation-cli.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-a-proxy-after-installation-cli_{context}"]
+= Configuring a proxy after installation using the CLI
+
+You can use the {product-title} (ROSA) CLI (`rosa`) to add a cluster-wide proxy configuration to an existing ROSA cluster in a Virtual Private Cloud (VPC).
+
+You can also use `rosa` to update an existing cluster-wide proxy configuration. For example, you might need to update the network address for the proxy or replace the additional trust bundle if any of the certificate authorities for the proxy expire.
+
+[IMPORTANT]
+====
+The cluster applies the proxy configuration to the control plane and compute nodes. While applying the configuration, each cluster node is temporarily placed in an unschedulable state and drained of its workloads. Each node is restarted as part of the process.
+====
+
+.Prerequisites
+
+* You have installed and configured the latest ROSA (`rosa`) and OpenShift (`oc`) CLIs on your installation host.
+* You have a ROSA cluster that is deployed in a VPC.
+
+.Procedure
+
+* Edit the cluster configuration to add or update the cluster-wide proxy details:
++
+[source,terminal]
+----
+$ rosa edit cluster \
+ --cluster $CLUSTER_NAME \
+ --additional-trust-bundle-file $CA_BUNDLE_FILE \
+ --http-proxy $HTTP_PROXY \
+ --https-proxy $HTTPS_PROXY
+----
```

Comment:
Now it allows to set --no-proxy filed to cluster via rosa cli but doesn't allow on UI.

Reply:
@yingzhanredhat thank you for your review and for highlighting this. Can you please expand on this topic? What explicitly needs to change in this PR in relation to this? Thanks!

Reply:
On review, I understand that what is required is to add the `--no-proxy` option to this CLI-based code block for ROSA. I can also add that in the similar code-block in `modules/configuring-a-proxy-during-installation-cli.adoc`. I will update this thread after I have tested and added that.

Reply:
@yingzhanredhat following on from a discussion within our team, I have created https://issues.redhat.com/browse/OSDOCS-3864 to track the requirement for the `--no-proxy` argument outside of this PR.

I will add that option as a separate exercise after this PR (47385) is merged, because it does not relate to the removal or replacement of the OCM CLI content that we are addressing here.